man/dracut.8.asc

```diff
@@ -530,6 +530,15 @@ will not be able to boot.
     Specifies the kernel image, which to include in the UEFI executable. The default is
     _/lib/modules/<KERNEL-VERSION>/vmlinuz_ or _/boot/vmlinuz-<KERNEL-VERSION>_
 
+**--cpio-reflink**::
+    Attempt to use the dracut-cpio binary, which optimizes archive creation for
+    copy-on-write filesystems by using the copy_file_range(2) syscall via Rust's
```

Comment:
I'd suggest to mention that copy_file_range(2) is available with kernel-5.3 and glibc-2.27

Reply:
The `copy_file_range()` syscall first appeared with linux 4.5 and the rust standard library invokes it via libc::syscall, so glibc version needn't be mentioned IMO. More important is the whether or not the rust standard library carries https://github.com/rust-lang/rust/pull/75272 , i.e. rust 1.50+.

I think including the term "reflink" in the feature could be a little misleading, as it's very filesystem specific. I think it might make more sense to use `--rapid-cpio`, `--accelerated-cpio` or similar.

Reply:
I've toyed with adding the following as a `configure` time check:
```
    rustc -V | awk -F '[ .]' \
        '$1 !~ /rustc/ {
             exit 1
         }
         $2 * 1000 + $3 <= 1050 {
             printf("Warning: %s < 1.50 does not support the copy_file_range() syscall\n", $1)
         }'
    uname -r | awk -F '.' \
        '$1 * 1000 + $2 < 4050 {
             print "Warning: Linux kernel < 4.5 does not support copy_file_range()"
         }'
```

I think it's a bit cumbersome, so will go with documentation for now.

Reply:
I've renamed the parameter `--enhanced-cpio`, to reflect that it won't necessarily result in reflinked cpio file data. The man page documents requirements for successful reflinks via `copy_file_range()`.